db/migrations/mysql/000003_create_user_groups.up.sql

```diff
@@ -0,0 +1,62 @@
+CREATE TABLE IF NOT EXISTS UserGroups (
+    Id varchar(26) NOT NULL,
+    Name varchar(64) DEFAULT NULL,
+    DisplayName varchar(128) DEFAULT NULL,
+    Description text,
+    Source varchar(64) DEFAULT NULL,
+    RemoteId varchar(64) DEFAULT NULL,
+    CreateAt bigint(20) DEFAULT NULL,
+    UpdateAt bigint(20) DEFAULT NULL,
+    DeleteAt bigint(20) DEFAULT NULL,
+    AllowReference tinyint(1) DEFAULT NULL,
+    PRIMARY KEY (Id),
+    UNIQUE KEY Name (Name),
+    UNIQUE KEY usergroups_source_remoteid_key (Source, RemoteId),
+    KEY idx_usergroups_remote_id (RemoteId),
+    KEY idx_usergroups_delete_at (DeleteAt)
+) ENGINE=InnoDB DEFAULT CHARSET=utf8mb4;
+
+SET @preparedStatement = (SELECT IF(
+    (
+        SELECT COUNT(*) FROM INFORMATION_SCHEMA.COLUMNS
+        WHERE table_name = 'UserGroups'
+        AND table_schema = DATABASE()
+        AND column_name = 'AllowReference'
+    ) > 0,
+    'SELECT 1',
+    'ALTER TABLE UserGroups ADD AllowReference tinyint(1);'
+));
+
+PREPARE alterIfNotExists FROM @preparedStatement;
+EXECUTE alterIfNotExists;
+DEALLOCATE PREPARE alterIfNotExists;
+
+SET @preparedStatement = (SELECT IF(
```

Comment:
Did those two failed in the CI checks as well or are we defaulting to create indexes both in the table and after for MySQL?